scripts/confset.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+function confset {
```

Comment:
If this script is going to be moved to `/usr/local/bin`, why not make it a command (instead of this function here) with the name `confset`, so that instead of running `source /usr/local/bin/confset.sh`, you can just run `confset some thing` in the script?